tests/core/conversations_worker/integration/__init__.py

```
@@ -379,3 +513,19 @@ def supabase_fx(request) -> SupabaseFixture:
             )
+
+    # Best-effort cleanup of RemoteToolCalls rows. The table has no DELETE RLS
+    # policy (rows normally leave only via the retention job), so this may be a
+    # no-op — that's fine for the isolated test cluster. Quiet on failure.
+    if fx._created_tool_calls and fx._relay_client is not None:
+        for tcid in fx._created_tool_calls:
+            try:
+                fx._relay_client.table("RemoteToolCalls").delete().eq(
+                    "id", tcid
+                ).execute()
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)